history.txt

```diff
@@ -1,6 +1,34 @@
 OMERO version history
 =====================
 
+5.1.0 (March 2015)
+------------------
+
+A full, production-ready release of OMERO 5.1.0; updating the Data Model to
+the January 2015 schema, including support for units and new more
+flexible user-added metadata; and introducing new user features, new supported
+formats and many fixes and performance improvements:
+
+-  support for units throughout the Data Model allowing for example, pixel
+   sizes for electron microscopy to be stored in nanometers rather than being
+   set as micrometers
+-  new, searchable key-value pairs annotations for adding experimental
+   metadata
+-  improved workflow for rendering settings in the UI and parity between the
+   clients
+-  import images to OMERO from ImageJ and save ROIs and overlays from ImageJ
+   to OMERO
+-  'Import As' for group owners as well as administrators
```

Comment:
My first impression when I read 'Import As' is that it must be like 'Save As', E.g. 'Import As OME-TIFF' or 'Import As SPW'. For users not familiar with 'Import As' (probably most, since only Admins have used this before), it might be clearer to say 'Import as another user' or just say "Group owners can now import data on behalf of other group members"?